mypy/checker.py

```diff
@@ -2148,11 +2148,18 @@ def check_override(
         """Check a method override with given signatures.
 
         Arguments:
-          override:  The signature of the overriding method.
-          original:  The signature of the original supertype method.
-          name:      The name of the subtype. This and the next argument are
-                     only used for generating error messages.
-          supertype: The name of the supertype.
+          override:                 The signature of the overriding method.
+          original:                 The signature of the original supertype method.
+          name:                     The name of the overriding method.
+                                    Used for generating error messages only.
```

Comment:
```suggestion
                                    Used primarily for generating error messages.
```
Dunders are ignored, see the `is_private` line